clients/batchValidator/factory/batchValidatorManagementFactory.go

```diff
@@ -0,0 +1,15 @@
+package factory
+
+import (
+	"github.com/ElrondNetwork/elrond-eth-bridge/clients"
+	batchValidatorManagement "github.com/ElrondNetwork/elrond-eth-bridge/clients/batchValidator"
+	"github.com/ElrondNetwork/elrond-eth-bridge/clients/batchValidator/disabled"
+)
+
+// CreateBatchValidator generates an implementation of BatchValidator
+func CreateBatchValidator(args batchValidatorManagement.ArgsBatchValidator, enabled bool) (clients.BatchValidator, error) {
```

Comment:
enabled bool not required to be passed as a sole parameter, you have it in the ArgsBatchValidator struct

Reply:
ArgsBatchValidator does not include the enabled bool value.